sdk/textanalytics/ai-text-analytics/src/transforms.ts

```diff
@@ -64,7 +63,7 @@ import {
   SentenceTarget,
   SentimentLROResult,
   SentimentTaskResult,
-  SingleClassificationDocumentResult,
+  ClassificationDocumentResult,
```

Comment:
Was going to ping our chat and see what everybody wants to call this model. I went with `ClassifyDocumentResult` to try to follow our verb-noun pattern, but I don't have strong opinions.

Reply:
I still have `CustomSingleLabelClassificationResult` and `CustomMultiLabelClassificationResult` where both has a `classifications` array property but you raise a good point. Maybe we should merge them now that the swagger does the same.

Reply:
For JS, I merged just the two types mentioned above but left the input and other output types unmerged. It is interesting though that Python is doing a deep merge, I remember Kareem pushed back on deep merging in the email thread I started after the CT archboard:
> Looking at the customization skills as a whole, single- and multi-label classification are completely different scenarios, so it makes much more sense to have them separated all up. Shipping them as separate skills in authoring and merging them in runtime will be super confusing (ruining the customer experience we are trying to enhance) and means we are shipping our organization not Language skills